parkerApp/src/Makefile

```diff
@@ -15,17 +15,23 @@ include $(TOP)/configure/CONFIG
 
 DBD += devPC6K.dbd
 DBD += ACRMotorSupport.dbd
+DBD += OEMMotorSupport.dbd
 DBD += ParkerSupport.dbd
 
 # Build a library wtih only ACR support for backwards compatibility
 LIBRARY_IOC += ACRMotor
+LIBRARY_IOC += OEMMotor
 
 ACRMotor_SRCS += ACRMotorDriver.cpp
-
 ACRMotor_LIBS += motor
 ACRMotor_LIBS += asyn
 ACRMotor_LIBS += $(EPICS_BASE_IOC_LIBS)
 
+OEMMotor_SRCS += OEMMotorDriver.cpp
+OEMMotor_LIBS += motor
+OEMMotor_LIBS += asyn
+OEMMotor_LIBS += $(EPICS_BASE_IOC_LIBS)
+
```

Comment:
You don't need to provide a library for OEM here, as there is no need for backward compatibility in this case. No one depends on it. They should depend on Parker directly.